app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,230 @@
+
+{% navtabs %}
+{% navtab Konnect UI %}
+In {% konnect_icon runtimes %} [**Runtime Manager**](https://cloud.konghq.com/us/runtime-manager), select a runtime group and follow these steps:
+
+1. Select **Gateway Services** from the side navigation bar, then **New Gateway Service**.
+
+1. From the **Add a Gateway Service** dialog, enter the following to create a new service:
+    * **Name:** classic_pizzas
+    * **Upstream URL:** `http://mockbin.org`
+    * Use the defaults for the remaining fields.
+1. Click **Save**. 
+1. To create an API product for your service, navigate to **API Products** in the sidebar, click **Add API Product** and enter `Classic Pizzas` in the **Product Name** field.
+1. Click **Publish to portal** from the **Actions** dropdown menu. This will allow developers to see your API product once you assign them to a team.
+1. Click **New Version** from the **Actions** dropdown menu and enter `v1` in the **Version Name** field and then click **Create**.
+1. Click the **v1** version and then click **Link** to add a service.
+1. Select a runtime from the **Select Runtime Group** dropdown menu, select **classic_pizzas** from the **Gateway Service** dropdown menu, and then click **Save**.
+1. Click the **Edit** icon next to the status of your product version and select **Published** from the **Edit version status** dialog and then click **Save**.
+
+{% endnavtab %}
+{% navtab API %}
+{:.note}
+> **Note:** Make sure you replace the placeholders in the following API requests with information specific to your environment. 
+
+1. Create the `classic_pizzas` service:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/<runtime-group-id>/core-entities/services \
+  -- data '{
+    "name": "classic_pizzas",
+    "retries": 5,
+    "protocol": "http",
+    "host": "example.com",
+    "port": 80,
+    "path": "/classic_pizzas",
+    "connect_timeout": 6000,
+    "write_timeout": 6000,
+    "read_timeout": 6000,
+    "tags": [
+      "user-level"
+    ],
+    "client_certificate": {
+      "id": "CERTIFICATE_ID"
+    },
+    "tls_verify": true,
+    "tls_verify_depth": null,
+    "ca_certificates": [
+      "CA_CERTIFICATE_ID"
+    ],
+    "enabled": true
+  }'
+```
+1. Create the `Classic Pizzas` API product:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/api-products \
+  --data '{
+    "id": "GATEWAY_SERVICE_ID",
+    "name": "Classic Pizzas",
+    "description": "API product for classic pizzas",
+    "portal_ids": "PORTAL_ID",
+    "labels": {
+        "env": "test"
+    }
+  }'
+```
+1. Create the `v1` version for the `Classic Pizzas` API product and publish it:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/api-products/<api-product-id>/product-versions \
+  --data '{
+    "id": "GATEWAY_SERVICE_ID",
+    "name": "v1",
+    "gateway_service": {
+        "runtime_group_id": "RUNTIME_GROUP_ID"
```

Comment:
```suggestion
        "runtime_group_id": "RUNTIME_GROUP_ID"
```

If this is needed you should say where they get it. I think this is from the output of the first request where you made the classic_pizzas service. 


According to the API docs the request should look like this

```
{
  "name": "v1",
  "publish_status": "unpublished",
  "deprecated": false,
  "gateway_service": {
    "runtime_group_id": "e4d9ebb1-26b4-426a-b00e-cb67044f3baf",
    "id": "09b4786a-3e48-4631-8f6b-62d1d8e1a7f3"
  }
}
```
Which has runtime_group_id and  id nested within gateway_service. I would copy that.

Reply:
> If this is needed you should say where they get it. I think this is from the output of the first request where you made the classic_pizzas service.

@Guaris It's used in the URL of the first request and from reading the specs, I don't think it's listed in the output, so they'd need to know this right away. Since this is just a tutorial and we want it to be opinionated, can I just recommend using "default" and do we need to point to a request that they can use to find the runtime group ID? I looked in the spec for one and couldn't find it.

Reply:
I think it's fine to just explain that you use the same `runtime-group-id` that you used in the first request.